resources/META-INF/plugin.xml

```diff
@@ -91,15 +95,16 @@
       </ul>
       ]]>
   </change-notes>
-  <version>0.9.9</version>
+  <version>1.0.0</version>
```

Comment:
Given the pattern, the version would likely be `0.10.0`

Reply:
+1

0.9.10 might even be more consistent with the version history thus far, but I like 0.10.0 since this breaks compatibility with older IntelliJ versions.

Reply:
+1 for versioning chnage.

Reply:
Thanks, will update to `0.10.0`

Reply:
@Sohamr this still needs to be updated